e2e/client/playwright/page-object-models/settings.ts

```diff
@@ -0,0 +1,26 @@
+import {Page, Locator} from '@playwright/test';
+import {s} from '../utils';
+
+export class Settings {
+    private page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+
+    async addFieldInContentProfile(contentProfile: string, tab: string, field: string): Promise<void> {
```

Comment:
there's already a class `ContentProfileSettings`. Move this method there. Improve naming to be `addFieldToContentProfile`.

Adjust the method so it can add multiple fields:
```
addFieldInContentProfile(contentProfile: string, fields: Array<{tabName: string; fieldId: string}>)
```

Also - utility methods should start and end in the same view. For example, this one starts in content profiles view where profiles are listed. When your utility method finished executing it should be in the same view. In your case it ends with one profile open and unsaved.